src/MessagePack/Formatters/TypelessFormatter.cs

```diff
@@ -113,6 +118,9 @@ static TypelessFormatter()
             Deserializers.TryAdd(typeof(object), _ => (object p1, ref MessagePackReader p2, MessagePackSerializerOptions p3) => new object());
         }
 
+#if NET8_0_OR_GREATER
```

Comment:
(nit) Why are these are sometimes guarded with `#if NET8` and sometimes not?

Reply:
Oh, just because I didn't switch to polyfill style attributes until I was midway through the work, and I missed removing the `#if` from some places. I removed it from here.